include/mbedtls/rsa.h

```diff
@@ -38,6 +38,60 @@
 #include "mbedtls/threading.h"
 #endif
 
+#if defined(PSA_CRYPTO_DRIVER_TEST)
+#include <psa/crypto_driver_common.h>
+#include <psa/crypto.h>
+
+typedef struct {
+    /* If non-null, on success, copy this to the output. */
+    void *forced_output;
+    size_t forced_output_length;
+    /* If not PSA_SUCCESS, return this error code instead of processing the
+     * function call. */
+    psa_status_t forced_status;
+    /* Count the amount of times one of the rsa driver functions is called. */
+    unsigned long hits;
+} mbedtls_test_driver_rsa_hooks_t;
+
+#define MBEDTLS_TEST_DRIVER_RSA_INIT { NULL, 0, PSA_SUCCESS, 0 }
+
+static inline mbedtls_test_driver_rsa_hooks_t
+     mbedtls_test_driver_rsa_hooks_init( void )
+{
+    const mbedtls_test_driver_rsa_hooks_t v = MBEDTLS_TEST_DRIVER_RSA_INIT;
+    return( v );
+}
+
+extern mbedtls_test_driver_rsa_hooks_t mbedtls_test_driver_rsa_hooks;
+
+psa_status_t mbedtls_test_transparent_asymmetric_encrypt(
+    const psa_key_attributes_t *attributes,
```

Comment:
You can compact this argument list a bit, it wastes a lot of space. There's a hard limit of 80 characters, that's usually only violated by comments, or some macros.

Reply:
fixed